### DISCUSSION TITLE
Subtitles are not synced properly with audio in SE

### DISCUSSION THREAD

**Discussion by Andrei-Iosifescu123** [openai/whisper#1077] [Q&A]
I noticed that whenever I'm using the Whisper speech recognition model, the subtitle appears before the speech. See the image below:
![image](https://user-images.githubusercontent.com/35394195/224505064-96a4b383-7920-403c-aa86-5dfa2f28aed4.png)
As you can see, `- They weren't configured for landing.` appears after `- The flaps are up too.`, but before they actually say it. Is there a way to make the subtitles appear during the speech, and not before or after?

**Comment by ryanheise** [comment]
The `--word_timestamps True` command line option might help improve the accuracy. Although this will also underline each word (this might become optional in the future: https://github.com/openai/whisper/discussions/1072#discussioncomment-5257759)
